pushnotifications/src/main/java/com/pusher/pushnotifications/fcm/MessagingService.kt

```diff
@@ -93,10 +93,10 @@ abstract class MessagingService: Service() {
           { token: String -> this.onNewToken(token) }
       )
 
-  override fun onBind(i: Intent?): IBinder = underlyingService.onBind(i)
+  override fun onBind(i: Intent?): IBinder = underlyingService.onBind(i!!)
 
   override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int =
-      underlyingService.onStartCommand(intent, flags, startId)
+      underlyingService.onStartCommand(intent!!, flags, startId)
```

Comment:
Excuse my Java ignorance, just curious what the `!!` operator is for?

Reply:
This is Kotlin operator !! to assert that an expression is non-null. This doesn't exist in Java.
New version of Kotlin is more scared of `null` than ever before so it is required here.